modules/installation/pages/client-requirements.adoc

```diff
@@ -3,38 +3,128 @@
+For {suse} operating systems, the version and SP level must be under general support (normal or LTSS) to be supported with {productname}.
 For details on supported product versions, see https://www.suse.com/lifecycle.
+For non-{suse} operating systems, including {rhel}, {centos}, and {oracle}, only the latest available version is under general support.
 ====
 
+
+
 [[mgr.supported.clients]]
 [cols="1,1,1,1", options="header"]
 .Supported Client Systems
 |===
-| Operating System | Architecture | Traditional Clients | Salt Clients
-| {sle} 15 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
-| {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
-| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
-| {sles}-ES 8 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sles}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sles}-ES 6 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sles} for SAP | {x86}_64, {ppc}                       | icon:check[role="green"]      | icon:check[role="green"]
-| {rhel} 8 | {x86}_64                                   | icon:question[role="gray"]    | icon:question[role="gray"]
-| {rhel} 7 | {x86}_64                                   | icon:check[role="green"]      | icon:check[role="green"]
-| {rhel} 6 | {x86}, {x86}_64                            | icon:check[role="green"]      | icon:check[role="green"]
-| CentOS 8 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| CentOS 7 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| CentOS 6 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| {opensuse} Leap 15 | {x86}_64                       | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 20.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 18.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+
+| Operating System
+| Architecture
+| Traditional Clients
+| Salt Clients
+
+| {sle} 15
+| {x86_64}, {ppc}, {ibmz}, ARM
+| {check}
+| {check}
+
+| {sle} 12
+| {x86_64}, {ppc}, {ibmz}, ARM
+| {check}
+| {check}
+
+| {sle} 11
+| {x86}, {x86_64}, IBM {ppc}, {ibmz}
+| {check}
+| {check}
+
+| {sles}-ES 8
+| {x86_64}
+| {check}
+| {check}
+
+| {sles}-ES 7
+| {x86_64}
+| {check}
+| {check}
+
+| {sles}-ES 6
+| {x86_64}
```

Comment:
All I can find about SLES-ES supported architectures is now at https://www.suse.com/media/data-sheet/sles_with_expanded_support_data_sheet.pdf, can't find anymore where it said that both x86 and x86_64 were supported but only for 6, while 7 and 8 are x86_64 only.

@mcalmer, can you confirm?

Reply:
Version 6 should have both x86 and x86_64. These are the channels which exists. There are no i386 channels for 7 and 8.